bigquery/tables.js

```diff
@@ -243,6 +266,29 @@ cli
   }, function (options) {
     program.exportTableToGCS(utils.pick(options, ['dataset', 'table', 'bucket', 'file', 'format', 'gzip']), utils.makeHandler());
   })
+  .command('insert <json_or_file> <dataset> <table>',
```

Comment:
Let's have `dataset` and `table` be the first two args in order to match all the other commands.